store/postgres/src/subgraphs.graphql

```diff
@@ -21,8 +21,16 @@ type SubgraphVersion @entity {
 type SubgraphDeployment @entity {
     id: ID! # Subgraph IPFS hash
     manifest: SubgraphManifest!
-    failed: Boolean!
+    failed: Boolean! @deprecated(reason: "Use `health`.")
+    health: Health!
     synced: Boolean!
+
+    "If the subgraph has failed, this is the error caused it"
+    fatalError: SubgraphError
+
+    "Sorted by block number, limited to 1000"
```

Comment:
Does that mean we will only ever store the latest 1000 errors for a subgraph, or that we will only return the 1000 latest errors when queried?

Reply:
The plan is to only store the _first_ 1000. Though we don't have non-fatal errors yet so we can still change that after this PR.